deps/build.jl

```diff
@@ -79,10 +79,9 @@ init_deps()
 end
 
 if is_apple()
-    if Pkg.installed("Homebrew") === nothing
-        error("Homebrew package not installed, please run Pkg.add(\"Homebrew\")")
-    end
     using Homebrew
+    Homebrew.update()
+    Homebrew.add("imagemagick")
```

Comment:
this is what the provider is supposed to do, but declaratively rather than imperatively